code/pilotfile/pilotfile.cpp

```diff
@@ -281,7 +281,7 @@ void pilotfile::update_stats_backout(scoring_struct *stats, bool training)
 
 			for (idx = 0; idx < list_size; idx++) {
 				if ( p_stats->ship_kills[idx].name.compare(Ship_info[i].name) == 0 ) {
-					j = i;
+					j = idx;
```

Comment:
I don't recall exactly where I got this code from (in 05adea0e), my best guess without an extensive search is that was something removed for 3.7.0. My current opinion is that I agree with @asarium, using idx makes more sense here as idx indexes to ship_kills and i indexes into m_okKills/Ship_info.